packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -821,6 +821,10 @@
                 {
                     "type": "cwsprChatMessageId"
                 },
+                {
+                    "type": "cwsprChatUserIntent",
+                    "required": false
+                },
```

Comment:
Do you have a plan/backlog item to port this to aws-toolkit-common?

Reply:
No, I missed the upstreaming part. Should I create an issue for tracking this?

Reply:
Yes, that is required for all metrics (with rare exceptions). All of the amazon q metrics in this file need to be upstreamed (and aligned with other IDEs), assuming they are still in use.

Reply:
Yes, after merging this you should treat it as high priority to move this metric to commons, especially if other IDEs need to use it.

Reply:
Got it, I will take that as an action item and work on it

Reply:
Update for aws-toolkit-common: https://github.com/aws/aws-toolkit-common/pull/864